### ISSUE TITLE
Identity loss term A2A typo in da_bidirectional_training.py? 

### ISSUE THREAD

**Issue by anlijuncn** [mlnotebook/domain_adapation_istn#1]
Hi Robinson, 

I am Lijun, a PhD stundent from NUS Singapore.  Nice work for domain adaption. 

I am reading your code, for indentity loss term A2A in da_unidirectional_training.py, is is a typo? 

For this [line](https://github.com/mlnotebook/domain_adapation_istn/blob/master/da_bidirectional_training.py#L243), shouldn't be `A2A = itn_B2A(A.detach())` instead of `A2A = itn_A2B(A.detach())` ?

Since `B2B = itn_A2B(B.detach())` in next B2B identity part [line](https://github.com/mlnotebook/domain_adapation_istn/blob/master/da_bidirectional_training.py#L247)

Based on my understanding, A2A and B2B should be dual right?

@mlnotebook

**Comment by mlnotebook** [comment]
Hi Lijun. Thank you for raising this issue.

You are correct! This error was introduced when the code was refactored before release. The error will be corrected in the next push.

I've also updated the title of the issue because it relates to the `bidirectional` rather than `unidirectional` script. Thank you again.
